manual/manual/cmds/intf-c.etex

```diff
@@ -986,15 +986,42 @@ Use the normal C array syntax instead.
 with the garbage collector using the "caml_register_global_root" function.
 \end{gcrule}
 
-Registration of a global variable "v" is achieved by calling
+The one may need to access values in OCaml heap from external code, for
+example when main loop of the program is implemented in C++. In this case the
+value of type "value" can be stored outside the OCaml heap for later use. In this
+subsection these external locations will be called \emph{global} values.
+
+Saving an OCaml value outside OCaml heap in a global value will work
+well when a piece of data is an unboxed integer and may work well when the it is a pointer
+outside OCaml heap. However, if a piece of data is a \emph{pointer to the OCaml heap} the
+one needs to register another location of this data in the OCaml runtime.
```

Comment:
"if … the one" ⇒ "if …, one"
"another location of this data" ⇒ "the location of this data"